lib/kaguya/core.ex

```diff
@@ -68,6 +68,17 @@ defmodule Kaguya.Core do
     {:noreply, %{socket: socket}}
   end
 
+  ## Active mode errors
+  def handle_info({:tcp_error, _socket, reason}, state) do
+    Logger.error fn -> "TCP Socket Error happend. Reason: #{reason}" end
```

Comment:
To ensure that formatting would happen only when log level is enabled.
This way formatting wouldn't happen if logging of error would be turned off.